content-src/components/ASRouterAdmin/ASRouterAdmin.jsx

```diff
@@ -805,11 +805,14 @@ export class CollapseToggle extends React.PureComponent {
   render() {
     const {props} = this;
     const {renderAdmin} = this;
-    const action = (this.state.collapsed || !renderAdmin) ? "Expand" : "Collapse";
+    const label = `${(this.state.collapsed || !renderAdmin) ? "Expand" : "Collapse"} devtools`;
     return (<React.Fragment>
       <a href="#devtools"
-        className={`asrouter-toggle asrouter-${action.toLowerCase()}`}
-        onClick={this.renderAdmin ? this.onCollapseToggle : null}>{action} Devtools</a>
+        title={label}
+        className={`asrouter-toggle ${this.state.collapsed ? "collapsed" : "expanded"}`}
```

Comment:
I think you need `this.state.collapsed || !renderAdmin` here like above for the label. Right now it's showing the expanded background on the first render.

Reply:
Or maybe that's intentional?

Reply:
gah. Ed beat me.